test/sslapitest.c

```diff
@@ -1878,6 +1879,9 @@ static int execute_test_session(int maxprot, int use_int_cache,
     if (s_options) {
         SSL_CTX_set_options(sctx, s_options);
     }
+    if (c_options) {
+        SSL_CTX_set_options(cctx, c_options);
+    }
```

Comment:
nit : useless brace on one line statement
I see you follow the style of previous statement.